README.md

```diff
@@ -127,6 +127,35 @@ c.LDAPAuthenticator.allowed_groups = [
 ]
 ```
 
+#### `LDAPAuthenticator.group_filter`
+
+The LDAP group search filter.
```

Comment:
We have `search_filter` (to identify a user) and this is described as `group search filter` - should we name it `group_search_filter` maybe?

Reply:
Done!